mission/landmarks/include/landmarks.h

```diff
@@ -0,0 +1,26 @@
+#include "ros/ros.h"
+#include "geometry_msgs/Point.h"
+#include <vortex_msgs/ObjectPosition.h>
+#include <map>
+#include <string>
+
+class Landmarks{
+    /*
+    To serve as an interface between the perception system and the control system,
+    an instance of this class receives object positions ("landmarks") published by 
+    the perception system on a ROS-topic. The object positions are stored in a map
+    before they are published on a ROS-topic which the control system is subscribed to. 
+    */
+public:
+    Landmarks();
+    void callback(vortex_msgs::ObjectPosition objPos);
```

Comment:
I think this callback needs a brief that explains what it does and the purpose of having it. The objPos argument also needs to be described (using the @param style)